src/fsharp/FSharp.Core.nuget/FSharp.Core.LatestNuget.nuspec

```diff
@@ -3,10 +3,10 @@
     <metadata>
         <!-- rename to FSharp.Core when done -->
         <id>FSharp.Core</id>
-        <title>FSharp.Core for F# 4.1</title>
-        <summary>FSharp.Core for F# 4.1</summary>
+        <title>FSharp.Core for F# 4.3</title>
+        <summary>FSharp.Core for F# 4.3</summary>
         <description>
-            FSharp.Core redistributables from Visual F# 4.1.0
+            FSharp.Core redistributables from Visual F# 4.3.0
```

Comment:
This text be something like "Visual F# SDK 4.3.0" (or, if you follow the advice above, "Visual F# SDK 4.1.3").  Using the official title of the MIcrosoft SDK would be best I suppose